subsys/usb/device/Kconfig

```diff
@@ -48,6 +48,21 @@ config USB_DEVICE_SN
 	  Serial Number String will be derived from
 	  Hardware Information Driver (HWINFO).
 
+config USB_DEVICE_CONFIGURATION_ENABLE
+	bool "USB configuration descriptor support"
+	default n
+	help
+	  Select 'y' to make the USB configuration descriptor available. The
+	  configuration descriptor is not available by default.
+
+if USB_DEVICE_CONFIGURATION_ENABLE
+config USB_DEVICE_CONFIGURATION
+	string "USB device configuration string"
+	default "Zephyr USB Sample"
```

Comment:
is this a reasonable default? not sure what type of string you'd expect to see in this descriptor but this seems like a good one for something else (like the product name)

Reply:
Something alongside "Sample Configuration" might be more reasonable because it would make easier for the user to determine where the string comes from (the string will be displayed somewhere on the host).

Reply:
I think this config may be overwriten by project if USB_DEVICE_CONFIGURATION_ENABLE is enabled. The default string is set to 'Zephyr USB Sample' to let users know this is a Zephyr sample device. Do you have any suggestions for this default string?

Reply:
@Chenhongren I think @tmon-nordic is suggesting to use "Sample Configuration"